<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportExemptionRecipesSetup</class>
 <widget class="QDialog" name="ReportExemptionRecipesSetup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="13" column="0">
    <widget class="QLabel" name="lblRecipeStatus">
     <property name="text">
      <string>Статус рецепта</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="CICDCodeEdit" name="edtMKBTo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string>a00.00; </string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0">
    <widget class="QCheckBox" name="chkOnlyVK">
     <property name="text">
      <string>Только по решению ВК</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="CICDCodeEdit" name="edtMKBFrom">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string>a00.00; </string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="5">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="5">
    <widget class="CGroupByExemptionRecipesComboBox" name="cmbSortBy"/>
   </item>
   <item row="17" column="4">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>ОК</string>
     </property>
    </widget>
   </item>
   <item row="14" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1" colspan="5">
    <widget class="CRecipeStatusComboBox" name="cmbRecipeStatus"/>
   </item>
   <item row="3" column="3" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblSocStatusType">
     <property name="text">
      <string>Код льготной категории граждан</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Источник финанисирования</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblMKB">
     <property name="text">
      <string>Коды диагнозов по МКБ</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblSortBy">
     <property name="text">
      <string>Группировать по</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="5">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="7" column="4">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>с</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lblBegDate_2">
     <property name="text">
      <string>Дата выписки рецепта:</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="5">
    <widget class="CRBComboBox" name="cmbSocStatusType"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>по</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <widget class="QLineEdit" name="edtClientId">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4">
    <widget class="QSpinBox" name="edtDrugCount">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="chkClientId">
     <property name="text">
      <string>Код пациента</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="chkDrugCount">
     <property name="text">
      <string>Количество единиц</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CGroupByExemptionRecipesComboBox</class>
   <extends>QComboBox</extends>
   <header>Reports/ComboBoxes</header>
  </customwidget>
  <customwidget>
   <class>CRecipeStatusComboBox</class>
   <extends>QComboBox</extends>
   <header>Reports/ComboBoxes</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>cmbSocStatusType</tabstop>
  <tabstop>edtMKBFrom</tabstop>
  <tabstop>edtMKBTo</tabstop>
  <tabstop>chkClientId</tabstop>
  <tabstop>edtClientId</tabstop>
  <tabstop>chkDrugCount</tabstop>
  <tabstop>edtDrugCount</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbRecipeStatus</tabstop>
  <tabstop>chkOnlyVK</tabstop>
  <tabstop>cmbSortBy</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ReportExemptionRecipesSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>ReportExemptionRecipesSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkClientId</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtClientId</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDrugCount</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtDrugCount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
